scipy/stats/stats.py

```diff
@@ -5173,6 +5174,113 @@ def friedmanchisquare(*args):
 
     return FriedmanchisquareResult(chisq, distributions.chi2.sf(chisq, k - 1))
 
+BrunnerMunzelResult = namedtuple('BrunnerMunzelResult', ('statistic', 'pvalue'))
+
+
+def brunnermunzel(x, y, alternative="two-sided", distribution="t"):
+    """
+    Computes the Brunner-Munzel test on samples x and y
+
+    Parameters
+    ----------
+    x, y : array_like
+        Array of samples, should be one-dimensional.
+    alternative :  'less', 'two-sided', or 'greater'
+        Whether to get the p-value for the one-sided hypothesis ('less'
+        or 'greater') or for the two-sided hypothesis ('two-sided').
+        Defaults value is two-sided.
+    distribution: 't' or 'g_norm'
```

Comment:
Given this is only a binary choice, `use_t=True` would be an alternative keyword (used in statsmodels)
However, `distribution` is better here because it would allow `permutation` as enhancement.